packages/adapter-node/src/temp/misc.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Return font face name from a file name
+ * Ex.: 'fonts/tital-one.woff' turns into 'Titan One'
+ * @param url - File url
+ */
+export function getFontFamilyName(url: string): string
+{
+    const ext = extname(url);
+    const name = basename(url, ext);
+
+    // Replace dashes by white spaces
+    const nameWithSpaces = name.replace(/(-|_)/g, ' ');
+
+    // Upper case first character of each word
+    const nameTitleCase = nameWithSpaces.toLowerCase()
+        .split(' ')
+        .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
+        .join(' ');
+
+    return nameTitleCase;
+}
+
+// Split a filename into [root, dir, basename, ext], unix version
+// 'root' is just a slash, or nothing.
+const splitPathRe
```

Comment:
You have node APIs here, there's no need to maintain this complicated regular expressions if you can use the path API for free.